README.md

```diff
@@ -223,6 +224,7 @@ These open source projects will help you to build a bot to automate your company
 
 ### Skype
 
+* [opsdroid](https://opsdroid.dev) (Python) + adapter
```

Comment:
Hey @Gideon357. Sorry for the delayed response. I can't find any information about the Skype adapter on the homepage. Do you mind sharing a link?